addons/website_sale_wishlist/static/src/js/website_sale.js

```diff
@@ -13,4 +22,282 @@ WebsiteSale.include({
         this._super(...arguments);
         $parent.find('button.o_wish_add').toggleClass('disabled', !isCombinationPossible);
     },
+    /**
+     * @constructor
+     */
+    init: function (parent) {
+        this._super.apply(this, arguments);
+        this.wishlistProductIDs = [];
+    },
+    /**
+     * Gets the current wishlist items.
+     * In editable mode, do nothing instead.
+     *
+     * @override
+     */
+    willStart: function () {
+        let self = this;
+        let def = this._super.apply(this, arguments);
+
+        let wishDef = $.get('/shop/wishlist', {
```

Comment:
The weird mix of modern javascript and old unnecessary jquery stuff (which isn't even really a gain) is rather strange.